build.gradle

```diff
@@ -460,6 +460,18 @@ task release() {
     }
 }
 
+// Publish to ~/.m2
+task releaseLocal() {
+    // This is necessary to upload the executable jar to bintray as well.
+    // It makes to run _bintrayRecordingCopy task after cli task.
```

Comment:
What happens if `_bintrayRecordingCopy` isn't called?

Reply:
`mustRunAfter` forces to run `cli` before the task run.
This is required when upload all-in-one jar to Bintray. 
But `releaseLocal` does not upload it to Bintray.
So I will remove the comment to avoid misleading.
We don't need cli binary itself for our purpose and no copy to local repository.
But keep consistency with `release`, I will keep `mustRunAfter`.
Thanks!